<launch>
    <arg name="map_info" />
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_info)" />
</launch>